support-frontend/assets/helpers/tracking/ophan.ts

```diff
@@ -1,6 +1,7 @@
 // ----- Imports ----- //
 import * as ophan from 'ophan';
 import type { NavigateFunction } from 'react-router';
+import type { NavigateOptions } from 'react-router-dom';
```

Comment:
Good to use the libraries type def here 👍

Reply:
Thanks